code/controllers/subsystems/voting/poll_types.dm

```diff
@@ -8,8 +8,8 @@
 	question = "End Shift?"
 	time = 90
 	choice_types = list(/datum/vote_choice/restart, /datum/vote_choice/countinue_round)
-	next_vote = 255 MINUTES //Minimum round length before it can be called for the first time
-	cooldown = 15 MINUTES //Cooldown is set to 15 mins as 1 hour is a bit much when things change so much in so little time + maxium 8 hour rounds means we should be a bit more forgiven.
+	next_vote = 300 MINUTES //Minimum round length before it can be called for the first time
+	cooldown = 30 MINUTES // Cooldown is set to 30 mins as people keep spamming roundend vote call
```

Comment:
Im ok with this but not ok with the min being raised to 5 hours!

Reply:
Four hour rounds have left little to no space for post-event RP, it's become a regular thing for the sake of a "fresh start" of which a high percentage of roundend voters don't even ready for another shift so not even their excuse of a "fresh, lag-less initialization" sustains itself. It's very detrimental to RP and one of the main reasons I grew disillusioned about having any sort of meaningful interaction after all the chaos had finally settled.